lua/ttt2/extensions/player.lua

```diff
@@ -5,22 +5,22 @@
 ---
 -- Resets the spawn position of all players.
 -- @realm server
-function player.ResetSpawnPosition()
+function player.ReSetLastSpawnPosition()
```

Comment:
```suggestion
function player.ResetLastSpawnPosition()
```

Reply:
the player library will be removed completely. I had talked about this with saibotk, so therefore this will no longer be important.